pyspark/bigdl/models/local_lenet/README.md

```diff
@@ -0,0 +1,18 @@
+# LeNet5 Model on MNIST
+
+LeNet5 is a classical CNN model used in digital number classification. For detail information,
+please refer to <http://yann.lecun.com/exdb/lenet/>.
+
+This example would show how to train and inference a LeNet model in pure local mode without using
+Spark local or Spark distributed cluster.
+ 
+## How to run this example:
+
+```
+pip install BigDL
```

Comment:
specify BigDL version?

Reply:
Or we can say after pip install (give link to pip install) and can directly run using `python` in command line.

Reply:
The latest version would be used by default. It would be worth getting rid of the version here in respect of maintenance.

Reply:
Fine